inc/widgets/author-bio.php

```diff
@@ -25,37 +25,57 @@ function __construct() {
 	function widget( $args, $instance ) {
 
 		global $post;
-		$author_id = $post->post_author;
 
-		$output = '';
+		extract( $args );
```

Comment:
https://vip.wordpress.com/documentation/vip/code-review-what-we-look-for/#using-extract